### DISCUSSION TITLE
Using Eigen matrix block to replace matrix reduce & increase

### DISCUSSION THREAD

**Discussion by DrChiZhang** [Xiangyu-Hu/SPHinXsys#519] [General]
@Shuaihao-Zhang @Xiangyu-Hu 
Shuaihao, in your present work of continuum dynamics, function of reduceTensor and increaseTensor is introduced. 
Would please check the eigen matrix block feature of 
https://eigen.tuxfamily.org/dox/group__TutorialBlockOperations.html
to simply the corresponding code?

**Comment by Shuaihao-Zhang** [comment]
@ChiZhangatTUM 
Thanks! I think this is a better solution.
I will update the code.

**Comment by DrChiZhang** [comment]
Done.
